addons/mail/static/src/discuss/call/common/call_participant_card.js

```diff
@@ -78,10 +79,21 @@ export class CallParticipantCard extends Component {
     }
 
     get showConnectionState() {
-        return Boolean(
-            this.isOfActiveCall &&
-                this.rtcSession?.peerConnection &&
-                !HIDDEN_CONNECTION_STATES.has(this.rtcSession.connectionState)
+        if (!this.rtcSession) {
+            return false;
+        }
+        if (!this.isOfActiveCall) {
+            return false;
+        }
+        if (HIDDEN_CONNECTION_STATES.has(this.rtcSession.connectionState)) {
+            return false;
+        }
```

Comment:
Nitpicking: could disjoin the conditionals with `||` to have less LOCs for a single `return false;`.